upgrading/automated_upgrades.adoc

```diff
@@ -328,8 +328,26 @@ $ ansible-playbook -i <path/to/inventory/file> \
     -e openshift_upgrade_nodes_label="region=group1"
 ----
 
-See xref:../admin_guide/manage_nodes.adoc#updating-labels-on-nodes[Managing Nodes] for more on node labels.
+The `openshift_upgrade_nodes_max_fail_percentage` variable allows you to specify
+how many nodes may fail in each batch. The percentage of failure must exceed
+your value before the playbook aborts the upgrade.
 
+The `openshift_upgrade_nodes_drain_timeout` variable allows you to specify how
+long to wait for the node to drain before forcefully killing the node.
+
+In this example, 10 nodes are upgraded at a time, the upgrade will abort if 20
+percent of the nodes fail, and there is a 600-second wait to drain the node.
+
+----
+$ ansible-playbook -i <path/to/inventory/file> \
+    </path/to/upgrade/playbook> \
+    e- openshift_upgrade_nodes_serial=10
+    e- openshift_upgrade_nodes_max_fail_percentage=20
+    e- openshift_upgrade_nodes_drain_timeout=600
+----
+
+See xref:../admin_guide/manage_nodes.adoc#updating-labels-on-nodes[Managing
+Nodes] for more on node labels.
```

Comment:
This sentence should probably still be connected to the above info on `openshift_upgrade_nodes_label`, possibly even turned into a NOTE box.